### ISSUE TITLE
Pub/Sub stops working after 1:05 minutes on ESP32

### ISSUE THREAD

**Issue by progxeno** [open62541/open62541#2642]
Hey all,
I am currently working on porting the open62541 for ESP32 (freertos/LwIP).
I managed to get a Publisher and a Subscriber running on the ESP32 but it always runs exactly 1 minute and 5 seconds. After this time it just stops.
For example the Publisher stops publishing although the task is still running. I tried different publishing intervals (100ms, 1s, 10s) but this has no effect. No matter how many messages are published, it always stops after this time.
I checked the free heap space and there is enough free.
The same code (Publisher and Subscriber) runs perfectly fine on a Raspberry Pi 3B+.
Anyone faces a similar issue?

 - [x] open62541 Version (release number or git tag): current master-branch
 - [x] Other OPC UA SDKs used (client or server): No other OPC UA SDKs are used!
 - [x] Operating system: freeRTOS/LwIP
 - [ ] Logs (with `UA_LOGLEVEL` set as low as necessary) attached
 - [ ] Wireshark network dump attached
 - [ ] Self-contained code example attached
 - [ ] Critical issue

**Comment by Pro** [comment]
Did you already check out #2511?
Also make sure that you are using current master branch.
There were similar issues with subscriptions especially on freertos.

**Comment by Marioam** [comment]
@Pro Yes I checked the problem with the server time but this doesn't seems to be the issue here.

**Comment by jpfr** [comment]
Can you print a log message in the publish callback.
If the log continues to be printed, it is likely a problem of the network stack.
